lib/index.iife.js

```diff
@@ -0,0 +1,23 @@
+(function(window) {
+  var VueDemi = {};
+  var Vue = window.Vue;
+  if (Vue) {
+    if (Vue.version.startsWith('2.')) {
```

Comment:
There is no `startsWith` in IE11, should check for `[0] === '2'` and `[1] === '.'`